README.md

```diff
@@ -16,3 +16,23 @@ The individual modules like "Reporting" require some setup and environment varia
 
 - **[Reporting](./docs/modules/reporting.md)**: Calculate aggregated reports and run queries on all data, accessible for external services for API integrations of systems
 - **[Export](./docs/modules/export.md)**: Template based file export API. Uses [carbone.io](https://carbone.io) as templating engine.
+
+_Modules have to be enabled via a feature flag in the environment config and may need additional environment variables (see module READMEs)._
+
+
+-----
+# Development
+
+## Configuration through environment variables
+Environment variables can be defined in an .env file using "snake case" variable names (see [examples/application.env](/docs/examples/application.env)).
+These are transformed automatically to be available in the code as nested properties
```

Comment:
Please confirm and/or elaborate, @tomwwinter .

The system and connections are not clear to me yet, that is why I want to add documentation for myself and other developers here